sdk/storage/storage-file-datalake/test/README.md

```diff
@@ -0,0 +1,22 @@
+# Testing
+
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+You can use existing Azure resources for the live tests, or generate new ones by using our [New-TestResources.ps1](https://github.com/Azure/azure-sdk-for-js/blob/master/eng/common/TestResources/New-TestResources.ps1) script, which will use an [ARM template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/test-resources.json) that already has all of the the necessary configurations.
+
+The Azure resources that are used by the tests in this project are:
+
+- Two general purpose, v2 [Azure Storage](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) account properly configured to test the Azure Storage Blob, Azure Storage Queue, Azure File Data Lake and Azure File Share services.
+
+You will also need to set the below environment variables to run the live tests:
+
+- `TEST_MODE`: Should have `live` assigned.
+- `ACCOUNT_NAME`: The Azure Storage account name.
```

Comment:
Prefixed with `DFS_` I think.

/cc - @jeremymeng @XiaoningLiu

Reply:
How though? None of the environment variables in https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/test-resources.json start with DFS_

Reply:
Tests internally look for the variables with DFS_ prefix, maybe the live tests are still not being executed as part of the nightly runs or the pipeline is not setup for the package.

Reply:
Seems like we should fix that!

Reply:
The file-datalake live test pipeline hasn't been set up yet?

Reply:
I don't see it here @jeremymeng 
![image](https://user-images.githubusercontent.com/10452642/80154361-42b3c680-8574-11ea-8819-75d85f9d66cd.png)